src/Excel.IO/ExcelExtensions.cs

```diff
@@ -26,48 +26,144 @@ public static string GetColumn(this Cell cell)
                 return string.Empty;
             }
 
-            var endIndex = 0;
-
-            for (int i = 0; i < cell.CellReference.Value.Length; i++)
-            {
-                if (char.IsLetter(cell.CellReference.Value[i]))
-                {
-                    endIndex = i + 1;
-                }
-            }
-
-            return cell.CellReference.Value.Substring(0, endIndex);
+            return cell.CellReference?.Value?[0..^-1];
```

Comment:
When I run any of the unit tests (including the one added as part of this PR) that does a read, it ends up with an ArgumentOutOfRangeException here.

Reply:
Weird, I'll investigate and sorry for that :(

Reply:
My bad I added an - and [0..^x] works like so: start from char 0 to last char -x where x is an int

Reply:
Again apologies because I added this over GitHub directly instead of thru ide so this last addition wasn't tested as it should've 😨

Reply:
Turns out that CellReference is (at least in some cases, don't know if in all cases) Row so can't just subtract 1 char because when it reaches for example 10 we will have A10 and the code would've subtracted 1 char making it A1.

I've reverted back to the commit before tiny refactoring commit